templates/main/about/privacy.html

```diff
@@ -20,119 +20,212 @@ <h1 class="uk-article-title hover-title" id="top">
                 <p>
                     We take every step to ensure that your data is used ethically and that includes making sure that
                     you know exactly what data we collect, and what we do with it. That means that instead of a
-                    bunch of legal mumbo-jumbo, we've provided this information in an easy, human-readable form below.
+                    bunch of legalese, we've provided this information in an easy, human-readable form below.
                 </p>
 
-                <h1 class="uk-article-title hover-title" id="data-collected">
-                    What We Collect
+                <table class="uk-table uk-table-divider uk-table-striped uk-table-small table-bordered">
+                    <thead>
+                        <tr class="thick-bottom-border">
+                            <th>What</th>
+                            <th class="uk-table-shrink">When</th>
+                            <th style="max-width: 30rem;">Usage</th>
+                            <th>Access</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>Discord user ID</td>
+                            <td class="uk-table-shrink"><code>self.accept()</code> run on Discord</td>
+                            <td style="max-width: 30rem;">Statistics, data association (infractions, code jam applications, etc)</td>
```

Comment:
aren't inline styles frowned upon?

Reply:
It depends, we do have them sprinkled liberally around. I do write actual CSS in the `style.css` if I think it'll be useful elsewhere, but often it's just specific styling for the one page.

Reply:
alright

Reply:
personally I do frown upon them.